charts/metering-ansible-operator/templates/crds/reportquery.crd.yaml

```diff
@@ -23,3 +23,103 @@ spec:
   - name: Age
     type: date
     JSONPath: .metadata.creationTimestamp
+
+  validation:
+  openAPIV3Schema:
+    required: ["spec"]
+    type: object
+    properties:
```

Comment:
I've been going with the `type: ...` -> `required` -> `properties` layout so far, and that style is what I've seen throughout most of the OLM stuff too.